### PR TITLE
Add feeds for relay-gateway app

### PR DESCRIPTION
**Pull request by mar3kk**

### CODE REVIEW

File: relay-gateway/Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=relay-gateway
+PKG_VERSION:=HEAD
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=$(TOPDIR)/../../packages/relay-gateway
```
Comment (rafcio2k) [inline relay-gateway/Makefile:9]: It should be https://github.com/CreatorKit/relay-gateway.git

Comment (nikhil-zinjurde-imgtec) [inline relay-gateway/Makefile:9]: I think keeping it the way it is at least on the dev branch will help developers to make changes locally and verify.  With the github link, the changes will need to be pushed first in order to do the verification.

This is something we have done for other gateway applications like button-led-controller, motion-led-controller. On master branch, we have ensured to use the github repo url

Comment (rafcio2k) [inline relay-gateway/Makefile:9]: OK, we could try this approach on dev branch

**Comment by abhijit-mahajani-imgtec** [comment]
Approved. only problem is we need to ensure having awa-0.2.1 (which is needed for project3) should not break existing projects 1 and 2, until we move to awa-0.2.1 on contiki.

**Pull request merged**